mmbexplorer/src/MMBFile.h

```diff
@@ -7,9 +7,11 @@ const unsigned char MMB_DISKATTRIBUTE_UNLOCKED    = 0x0F; // Unlocked
 const unsigned char MMB_DISKATTRIBUTE_LOCKED      = 0x00; // Locked
 const size_t        MMB_MAXDISKNAMELENGTH         = 12;   // As per the Acorn Disk Filing System
 const size_t        MMB_MAXNUMBEROFDISKS          = 511;
+const size_t        MMB_MAXNUMBEROFDISKS2         = 511*15;
```

Comment:
Shouldn't this be 511*16? That would make 8176 disks and not 7665. I got to this trying to create a maximum size MMB file.

Reply:
yep you are correct. I have fixed this